shared/profile/index.desktop.js

```diff
@@ -262,13 +262,20 @@ class ProfileRender extends PureComponent<void, Props, State> {
       <Box style={styleOuterContainer}>
         <Box style={{...styleScrollHeaderBg, backgroundColor: trackerStateColors.header.background}} />
         <Box style={{...styleScrollHeaderCover, backgroundColor: trackerStateColors.header.background}} />
-        {this.props.onBack &&
-          <BackButton
-            onClick={this.props.onBack}
-            style={{position: 'absolute', left: 14, top: 16, zIndex: 12}}
-            textStyle={{color: globalColors.white}}
-            iconStyle={{color: globalColors.white}}
-          />}
+        <Box style={{...globalStyles.flexBoxColumn}}>
+          {this.props.onBack &&
+            <BackButton
+              onClick={this.props.onBack}
+              style={{position: 'absolute', left: 14, top: 16, zIndex: 12}}
```

Comment:
instead of 12/13 we should have something like backZindex and searchContanerZindex = backZindex + 1 or something